pkgs/misc/ghostscript/default.nix

```diff
@@ -26,6 +27,7 @@ let
       mv -v * "$out/"
     '';
   };
+  dvips = texlive.combine { inherit (texlive) scheme-infraonly dvips; };
```

Comment:
I don't think this is helpful. `dvi` files don't contain any embedded fonts. dvips looks up font files (and other external files) using kpathsea and embeds them into the `ps` output. But if you take dvips from such a minimal texlive installation, this font lookup will always fail! So you won't be able to convert any nonempty file (ok, there might be some edge cases using only specials that work, but these won't appear in the real world ever).

Of course using `texlive.combined.scheme-full` here is ridiculous, so I think it's better to not include dvips in the wrapper at all. Then the user can at least provide their own dvips program (with the neccessary fonts etc).